### PR TITLE
New package: Opencoarrays-2.9.2

### PR DESCRIPTION
**Pull request by ramos**

### CODE REVIEW

**Comment by kawaiiamber** [comment]
Get rid of line 15 to pass Link check.

File: srcpkgs/OpenCoarrays/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'OpenCoarrays'
+pkgname=OpenCoarrays
+version=2.9.2
+revision=1
+build_style=cmake
+short_desc="OpenCoarray fortran PAGS paralellization library"
+maintainer="Alberto Ramos <alberto.ramos@ific.uv.es>"
+hostdepends="gcc gcc-fortran openmpi openmpi-devel cmake"
```
Comment (kawaiiamber) [inline srcpkgs/OpenCoarrays/template:8]: `hostmakedepends`, -devel things should go in `makedepends`. The `build_style` is `cmake`, no need to include `cmake` in `hostmakedepends` or in `makedepends`. Looking at the project, you might want `openmpi-devel` for the development files in `makedepends`.

File: srcpkgs/OpenCoarrays/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+makedepends="gcc gcc-fortran openmpi openmpi-devel cmake"
+depends="gcc gcc-fortran openmpi openmpi-devel cmake"
+license="GPL-3.0-or-later"
```
Comment (kawaiiamber) [inline srcpkgs/OpenCoarrays/template:11]: The license is actually a BSD 3 Clause license, change to `license="BSD-3-Clause"`, and add
```
post_install() {
    vlicense LICENSE
}
```
but with a tab before vlicense, not spaces.

**Review by kawaiiamber — CHANGES_REQUESTED** [review]

File: srcpkgs/OpenCoarrays/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+homepage="http://www.opencoarrays.org/"
+distfiles="https://github.com/sourceryinstitute/OpenCoarrays/releases/download/2.9.2/OpenCoarrays-${version}.tar.gz"
```
Comment (kawaiiamber) [inline srcpkgs/OpenCoarrays/template:13]: You hardcoded `2.9.2` in one spot in the URL yet expanded `${version}` in another? It's recommended to expand `${version}` when possible to make maintaining easier. Try switching `distfiles` to `distfiles="https://github.com/sourceryinstitute/OpenCoarrays/archive/${version}.tar.gz"` and match the checksum.

**Comment by kawaiiamber** [comment]
There are three commits, there should only be one. Try squashing your commits with rebase: `git rebase -i HEAD~3` and squash the last two, having the commit message just be "New package: OpenCoarrays-2.9.2", just use `amend` to commit future changes.
```
git commit --amend
git push origin master -f
```
One thing I might recommend while I'm on git things, I would recommend making a specific branch on your fork, e.g.
```
git checkout -b opencoarrays
<add your template>
git add -A
git commit -m "New package: OpenCoarrays-2.9.2"
git push origin opencoarrays
```
then, on this PR, switch to request merging that branch from your fork to the upstream master branch.

**Comment by kawaiiamber** [comment]
Haven't tested, but, from what I've read on upstream's documentation, [this](https://pastebin.com/z4ea1hEu) template might be helpful as a start point.

**Comment by kawaiiamber** [comment]
I tried testing on aarch64-musl, and found that:
```
=> OpenCoarrays-2.9.2_1: building [cmake] for aarch64-musl...
   [host] gcc-fortran-9.3.0_9: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [host] openmpi-4.1.0_1: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [host] cmake-3.19.3_1: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [target] libgfortran-devel-9.3.0_9: found (https://alpha.de.repo.voidlinux.org/current/aarch64)
   [target] openmpi-devel-4.1.0_1: not found
```
`openmpi` isn't on cross platform. I have an updated template [here](https://pastebin.com/pZQzpSQB). I don't know much about fortran, though.

**Comment by ramos** [comment]
Ok, Many thank

Not sure about the conflicts. I have checked that this builds ok in my system with your new template.

Many thanks!

**Comment by kawaiiamber** [comment]
It looks like you pulled all changes from latest upstream master branch - over 700 commits! You might want to do a couple of things:
1. Add an upstream remote: `git remote add upstream https://github.com/void-linux/void-packages.git`
2. Rebase your branch against upstream master: `git pull --rebase upstream master`
3. Add a branch specifically for your template: `git checkout -b opencoarrays`, then add your template and push there: `git push origin opencoarrays`, switch this PR to merge from that branch rather than your forked master branch.

**Pull request closed without merging**